src/pages/employee/experience.tsx

```diff
@@ -1,57 +1,73 @@
-  }
-const Experience:React.FC<ExperienceProps> = ({setTab}) => {
-  return (
-    <div className='border rounded-xl mt-6 flex flex-col justify-end items-end'>
-    <div className='pt-6 px-6 grid grid-cols-2 gap-6 w-full'>
-      <div>
-          <label htmlFor="employer">
-              Employer
-          </label>
-          <Input
-          type='text' placeholder='Employer'
-          />
-      </div>
-      <div>
-          <label htmlFor="jobtitle">
-              Job Title
-          </label>
-          <Input
-          type='text' placeholder='Job Title'
-          />
-      </div>
-      <div>
-          <label htmlFor="from">
-              From
-          </label>
-          <DatePicker/>
-      </div>
-      <div>
-          <label htmlFor="to">
-              To
-          </label>
-          <DatePicker/>
+    setTab: React.Dispatch<React.SetStateAction<string>>;
+}
 
-      </div>
-      <div>
-          <label htmlFor="duration">
-              Duration
-          </label>
-          <Input
-          type='text' placeholder='Duration'
-          />
-      </div>
-    </div>
-    <Button type='filled' state='primary' onClick={()=>{setTab('document')}} className='m-6 px-9 py-3'>Next</Button>
+const Experience: React.FC<ExperienceProps> = ({ setTab }) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const handleCloseModal = () => setIsOpen(false);
+    const handleOpenModal = () => setIsOpen(true);
 
-  </div>
-  )
-}
+    const experienceData = [
+        {
+            'Employer': 'Catalysts Reachout',
+            'Job Title': 'Frontend Developer',
+            'From Date': '20-12-2023',
+            'To Date': '12-01-2024',
+            'Duration': '4 months'
+        },
+        {
+            'Employer': 'Catalysts Reachout',
+            'Job Title': 'Frontend Developer',
+            'From Date': '20-12-2023',
+            'To Date': '12-01-2024',
+            'Duration': '4 months'
+        },
+        {
+            'Employer': 'Catalysts Reachout',
+            'Job Title': 'Frontend Developer',
+            'From Date': '20-12-2023',
+            'To Date': '12-01-2024',
+            'Duration': '4 months'
+        },
+    ];
+
+    return (
+        <div >
+            <EmployeExperienceModal
+                isOpen={isOpen}
+                onClose={handleCloseModal}
+                onSubmit={handleCloseModal}
+            />
+            <div className='flex justify-end'>
+                <Button type='filled' state='primary'
+                    className="px-[10px] py-[6px] rounded"
+                    onClick={handleOpenModal}
+                >
+                    <FaPlus className='mr-[4px]' />
+                    Add more
+                </Button>
+            </div>
+            {experienceData.map((experience, index) => (
+                <div key={index} className='border rounded-xl flex flex-col my-4 justify-end items-end'>
```

Comment:
make this whole section one single component

 `<div key={index} className='border rounded-xl flex flex-col my-4 justify-end items-end'>
                    <div className='p-4 grid grid-cols-3 w-full'>
                        {Object.entries(experience).map(([key, value]) => (
                            <EmployeeCard
                                key={key}
                                label={key}
                                value={value}
                            />
                        ))}
                    </div>
                </div>`
                
          such that      
          {experienceData.map((experience, index) => (
          <component data={experience} />
          ))}